sources/packages/web/src/components/common/InstitutionUserSummary.vue

```diff
@@ -44,6 +45,7 @@
       @page="paginationAndSortEvent($event)"
       @sort="paginationAndSortEvent($event)"
       :loading="loading"
+      breakpoint="1250px"
```

Comment:
just a thought. how about adding the breakpoint as a constant

Reply:
Not sure. It is a PrimeVue thing due to the bad way that it responds while reducing the table. It must be defined per table because it depends on the table width and number of columns. I would say that it is a temporary solution that must be applied per table.

Reply:
oh..its a primevue componet. never mind. ignore the comment